Tensile/Common.py

```diff
@@ -396,6 +396,11 @@ def getArchitectureName(gfxName):
     "WaveSeparateGlobalReadA":    [ 0, 1 ],
     "WaveSeparateGlobalReadB":    [ 0, 1 ],
 
+    # Splits global read addresses within a wave into a number of smaller groups
+    # The default value of 1 reads the most contiguous elements across lanes,
+    # but higher values may help avoid bank conflicts
+    "SplitGlobalRead":            [1, 2, 4, 8, 16, 32],
```

Comment:
I believe values 16, 32 are unnecessary choosing those values results into low memory load efficiency.

Reply:
Yea, mainly had them for testing. I've put in a change to remove those values since they'd be impractical.